update dogstatsd docs (#16740)
DataDog/documentation: content/en/developers/dogstatsd/_index.md

              gcr.io/datadoghq/agent:latest
```

> If you need to change the port used to collect StatsD metrics, use the `DD_DOGSTATSD_PORT="<NEW_DOGSTATSD_PORT>` environment variable. You can also configure DogStatsD to use a [Unix domain socket][1]

Reviewer: ```suggestion
If you need to change the port used to collect StatsD metrics, use the `DD_DOGSTATSD_PORT="<NEW_DOGSTATSD_PORT>` environment variable. You can also configure DogStatsD to use a [Unix domain socket][1].
```

#### Origin detection over UDP

> Origin detection is supported in Agent 6.10.0+ and allows DogStatsD to detect where the container metrics come from, and tag metrics automatically. When this mode is enabled, all metrics received through UDP are tagged by the same pod tags as Autodiscovery metrics.

Reviewer: ```suggestion
Origin detection is supported in Agent v6.10.0+, and allows DogStatsD to detect where the container metrics come from and automatically tag metrics. When this mode is enabled, all metrics received through UDP are tagged by the same pod tags as Autodiscovery metrics.
```

> Origin detection in non-Kubernetes environments is based on an extension of the dogstatsd protocol: [Datagram Format and Shell Usage](https://docs.datadoghq.com/developers/dogstatsd/datagram_shell/?tab=metrics#dogstatsd-protocol-v12). To enable the feature in the agent the environment variable `DD_DOGSTATSD_ORIGIN_DETECTION_CLIENT=true` needs to be set.

Reviewer: ```suggestion
Origin detection in non-Kubernetes environments is based on an extension of the DogStatsD protocol in [Datagram Format and Shell Usage][2]. To enable the feature in the Agent, set the `DD_DOGSTATSD_ORIGIN_DETECTION_CLIENT` environment variable to `true`.
```

> **Note:** Origin Detection is not supported for Fargate environments.

Reviewer 2: ```suggestion
**Note**: Origin Detection is not supported for Fargate environments.
```

> **Note**: Origin Detection is not supported for Fargate environments.

Reviewer: ```suggestion
**Note**: Origin detection is not supported for Fargate environments.
```

> [1]: /developers/dogstatsd/unix_socket/

Reviewer: ```suggestion
[1]: /developers/dogstatsd/unix_socket/
[2]: /developers/dogstatsd/datagram_shell/?tab=metrics#dogstatsd-protocol-v12
```
